recipes/icqsol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: icqsol
+  version: "0.3.20"
+
+about:
+  home: 'https://github.com/pletzer/icqsol'
+  summary: 'A collection of utilities for constructing complex geometries from primitive shapes'
+  license: MIT
+
+source:
+  git_url: https://github.com/pletzer/icqsol.git
+  git_rev: 05b229c
+
+build:
+  number: 0
+
+test:
+  # Python imports
+  imports:
+    - csg
+    - icqsol
+    - numpy
+    - triangle
+    - vtk
+
+requirements:
+  build:
+    - cmake
+    - numpy
+    - python
+    - pycsg ==0.3.12
```

Comment:
@bgruening Do you think that since the recipes for pycsg and pytriangle specify git-revs that this is sufficient, and that I can eliminate the pinned versions in this recipe?  If so, I will.  Thanks!